lib/matplotlib/lines.py

```diff
@@ -592,6 +599,61 @@ def get_markevery(self):
         """return the markevery setting"""
         return self._markevery
 
+    @property
+    def downsample(self):
+        """Set the downsample property to subsample the plotted line segments.
```

Comment:
The current docstring is a mixture of our old and new style of docstring. Can you please stick to the new style? (I'm adding a couple notes to explain what I mean).